src/jupyter/jupyterlabs-pyenv/setup_pyenv.sh

```diff
@@ -0,0 +1,60 @@
+#!/encs/bin/tcsh
+
+# Prevent running on submit nodes
+if ( `hostname -s` =~ *submit* ) then
+    echo "String 'submit' found in hostname. This script should be run in srun or salloc. Exiting."
+    exit 1
+fi
```

Comment:
```suggestion
endif
```

per comment from @carlos-encs ; please retest with `tcsh -n setup_pyenv.sh` before running